SIS/clarin/data/recommendations/CLARIN-IS-recommendation.xml

```diff
@@ -22,16 +22,18 @@
     </header>
     <info><!--use p, ul, ol elements here--></info>
     <formats><!--
-            This template will assist in adjusting/creating a recommendations document if you don't
-            have access to a schema-aware XML editor. Copy the XML fragment, 
-            *  adjust the format ID (see https://clarin.ids-mannheim.de/standards/views/list-formats.xq ),
-            *  adjust the domain (https://clarin.ids-mannheim.de/standards/views/list-domains.xq ),
-            *  adjust the recommendation level ({ recommended, acceptable, discouraged }),
-            *  repeat as necessary;
-            *  submit the result as a PR against https://github.com/clarin-eric/standards/tree/formats .
-
-       -->
-        
+        <p>This page lists the file formats which are recommended when depositing in CLARIN-IS repository.</p>
```

Comment:
This is added outside of the relevant XML structure and since I have other items in the pull pipeline, I will copy these changes straight into their proper place in the recommendations, attribute them to you, and close this PR.